tests/test_phase5_rebuild_ete.py

```
@@ -0,0 +1,648 @@
+    assert re.search(
+        r"ac\.storage\s*\(\s*[\"'][\w_]*[Pp]laced",
+        entry_src,
+    ), 'autoPlaceOnce must persist via ac.storage("...placed...", ...)'
+
+
+# ---------------------------------------------------------------------------
+# ETE-08: hud.lua uses ui.dwriteDrawText / pushDWriteFont, not ui.text
+# ---------------------------------------------------------------------------
+
+
+def test_ete08_hud_uses_absolute_positioned_drawing(lua):
```

Ruff (0.15.9)

[warning] Unused function argument: `lua`

(ARG001)